pheweb/serve/static/variant.js

```diff
@@ -280,13 +282,18 @@ LocusZoom.TransformationFunctions.set("percent", function(x) {
 	    if (af_popmax && !isNaN(parseFloat(af_popmax)) && isFinite(af_popmax)) {
 		af_popmax = af_popmax.toExponential(1)
 	    }
-	    $('#gnomad').html('<p style="text-decoration: underline;" title="' + _.template($('#gnomad-tooltip-template').html())({v:window.variant}) + '" data-toggle="tooltip">AF in gnomAD genomes 2.1: FIN ' + af_fin + ', POPMAX ' + af_popmax + ', FIN enrichment vs. NFEE: ' + window.variant.gnomad.fin_enrichment + '</p>')
+	    if (window.browser == 'FINNGEN') {
+		$('#gnomad').html('<p style="text-decoration: underline;" title="' + _.template($('#gnomad-tooltip-template').html())({v:window.variant}) + '" data-toggle="tooltip">AF in gnomAD genomes 2.1: FIN ' + af_fin + ', POPMAX ' + af_popmax + ', FIN enrichment vs. NFEE: ' + window.variant.gnomad.fin_enrichment + '</p>')
+	    } else {
+		$('#gnomad').html('<p>Frequency in gnomAD genomes 2.1.1: ' + Object.keys(window.variant.gnomad).filter(g => g.startsWith('AF')).map(g => g + ': ' + window.variant.gnomad[g]).join(', ') + '</p>')
+	    }
             $('#gnomad p').css('margin-bottom', '0');
 	})
     }
 })();
 
 (function() {
+    if (window.browser != 'FINNGEN') return
```

Comment:
could you check if there is INFO annots and show that... or maybe add also this to conf if it's shown or not. less config if automatic.